.config-schema.json

```diff
@@ -186,29 +186,6 @@
                 }
             }
         },
-        "otel": {
-            "type": "object",
-            "properties": {
-                "metrics": {
-                    "type": "object",
-                    "properties": {
-                        "endpoint": {
-                            "description": "Remote OTEL collector endpoint",
-                            "type": "string",
-                            "default": "0.0.0.0:4317",
-                            "x-env-variable": "OPENFGA_OTEL_METRICS_ENDPOINT"
-                        },
-                        "protocol": {
-                            "description": "Protocol used to send metrics to the collector",
-                            "type": "string",
-                            "enum": ["grpc", "http"],
-                            "default": "grpc",
-                            "x-env-variable": "OPENFGA_OTEL_METRICS_PROTOCOL"
-                        }
-                    }
-                }
-            }
-        },
```

Comment:
don't see them mentioned in https://github.com/openfga/openfga/blob/58591cd8dc6de6de599397216163485f159fae6e/cmd/run/flags.go#

Reply:
That is correct, we dropped the OTEL metrics integration altogether, so this can be removed. It's residual 👍